transformations/transformations/data_offload.py

```diff
@@ -27,13 +27,17 @@ class DataOffloadTransformation(Transformation):
     ----------
     remove_openmp : bool
         Remove any existing OpenMP pragmas inside the marked region.
+    assume_deviceptr : bool
```

Comment:
Docstring and actual argument are out of sync here now. Can we call the argument `assume_deviceptr`?